### PR TITLE
PPSS-44146 Release application browser 4.0.4

### PR DESCRIPTION
**Pull request by caeduroa**
Releasing 4.0.4 version.

### CODE REVIEW

**Comment by caeduroa** [comment]
Hi @kevinmprince and @ARobertsAtPros, can you please help me taking a look to this PR to approve and merge it?

File: build.gradle [hunk 1]
```diff
@@ -63,7 +76,7 @@ task getLicenseAndMake(dependsOn: ['getLicense', 'make']) {
 
 wrapper {
    gradleVersion = '6.2.2'
-   distributionUrl = "${repositoryURL}/plugins-releases/org/gradle/gradle/${gradleVersion}/gradle-${gradleVersion}-bin.zip"
+   distributionUrl = "http://cache.devops.pros.com/services.gradle.org/distributions/gradle-${gradleVersion}-bin.zip"
```
Comment (kevinmprince) [inline build.gradle]: It looks like this used to be configurable. I'm not sure if referencing "http://cache.devops.pros.com/services" qualifies this as being open source.

File: build.gradle [hunk 2]
```diff
@@ -20,41 +20,54 @@
 
 import groovy.json.JsonBuilder
 import groovy.json.JsonSlurper
+import org.apache.tools.ant.taskdefs.condition.Os
 
 buildscript {
     repositories {
         maven {
-            url "${repositoryURL}/repo"
+            url "${repositoryURL}/artifactory/maven-onprem-dev"
```
Comment (kevinmprince) [inline build.gradle]: Not sure why we expose this now.
I suspect gradle isn't needed at all outside of our internal build system. So, the open source aspect of this product is not affected by this, but I would be happier if it was not published to public github.

File: package.json [hunk 3]
```diff
@@ -59,6 +60,9 @@
     },
     "devDependencies": {
         "@babel/parser": "7.10.4",
+        "@babel/plugin-transform-modules-amd": "7.10.4",
+		"@babel/plugin-transform-modules-umd": "7.10.4",
```
Comment (kevinmprince) [inline package.json]: minor detail, the formatting if off.

**Review by kevinmprince — COMMENTED** [review]
I approve the changes, but request Adam confirm the references to maven-onprem-dev and devops.pros.com are acceptable

File: build.gradle (hunk 2 above)
Comment (caeduroa) [reply build.gradle:28]: I'll create a ticket to remove gradle from App Browser for 4.05. Can we release as it is for now?

File: build.gradle (hunk 1 above)
Comment (caeduroa) [reply build.gradle]: I removed that part and I changed it by a variable.

File: package.json (hunk 3 above)
Comment (caeduroa) [reply package.json]: Done.

File: build.gradle (hunk 1 above)
Comment (caeduroa) [reply build.gradle]: I just removed gradle.

File: build.gradle (hunk 2 above)
Comment (caeduroa) [reply build.gradle]: I just removed gradle.

File: .npmrc [hunk 4]
```diff
@@ -1,2 +1,4 @@
 cache=build/npmcache
+registry=REPOSITORY_URL/artifactory/api/npm/npm-onprem-dev/
```
Comment (kevinmprince) [inline .npmrc]: this should not be needed here

**Review by kevinmprince — APPROVED** [review]

**Pull request merged**